src/App.css

```diff
@@ -1,38 +1,3 @@
 .App {
   text-align: center;
 }
```

Comment:
Can you please move the alignment to the component so other components are not affected?